packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -185,6 +189,12 @@ marketVolConfigurator18 = "<%= settings.storkExecutor18 %>"
 marketVolConfigurator19 = "<%= settings.storkExecutor19 %>"
 marketVolConfigurator20 = "<%= settings.storkExecutor20 %>"
 
+# TODO: add proper addresses
+[var.share_token_roles]
+rseliniSubscriber1 = "0x0000000000000000000000000000000000000000"
+rseliniRedeemer1 = "0x0000000000000000000000000000000000000000"
+rseliniCustodian1 = "0x0000000000000000000000000000000000000000"
```

Comment:
for he custodian let's use this https://safe.reya.network/home?safe=reya:0x75cfe7F41953cDfeA30C9F6A0BceC6BAA3dA71B0

Reply:
created new one

Reply:
sure